scripts/Source/sslThreadController.psc

```diff
@@ -637,7 +637,7 @@ state Animating
 		UnregisterForUpdate()
 		ApplyFade()
 		GoToState("Refresh")
-		SyncEvent(kRefreshActor, 10.0)
+		SyncEvent(kRefreshActor, 15.0)
```

Comment:
Adjusting Wait Times for SyncEvent

After all the changes on the scripts become necessary adjust the value to prevent issues related with the Update event